lib/rtc_audio_observer.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:typed_data';
```

Comment:
I'd rather move it to src/* and export it in rtc_engine

Reply:
Ok, but imo the god class approach is really poor idea and breaking SRP